hpe_3par_kubernetes_manager.py

```diff
@@ -2162,6 +2240,28 @@ def corden_node(name):
         raise e
 
 
+def stop_kubelet(name):
+    try:
+        command = "systemctl stop kubelet"
+        logging.getLogger().info(command)
+        output = get_command_output(name, command)

```

Comment:
Please ensure that you will receive the expected output.

Reply:
Added status_kubelet which checks if the kubelet service was stopped/started